src/Layout/QueryParamContext.tsx

```diff
@@ -294,19 +307,13 @@ export default function QueryParamProvider({ children }: any) {
   }, [router.query.to, country]);
 
   async function loadConfig() {
-    let userLang;
-    if (localStorage) {
-      userLang = localStorage.getItem("language") || "en";
-    } else {
-      userLang = "en";
-    }
     try {
       const requestParams = {
         url: `/app/config`,
         setshowErrorCard,
         shouldQueryParamAdd: false,
       };
-      const config: any = await apiRequest(requestParams);
+      const config: LoadConfig = await apiRequest(requestParams);
```

Comment:
This looks like axios response, we don't need TS declaration for axios response.
We only need TS declaration of data that is been returned.

Please remove TS declaration related to axios response